### PR TITLE
Pawelka/nupkg renames

### PR DESCRIPTION
**Pull request by moozzyk**
Server:
 - Rename Microsoft.AspNetCore.SignalR -> Microsoft.AspNetCore.SignalR.Core
 - Rename Microsoft.AspNetCore.SignalR.Http -> Microsoft.AspNetCore.SignalR 
 
Client:
 - Rename Microsoft.AspNetCore.SignalR.Client -> Microsoft.AspNetCore.SignalR.Client.Core
 - Create new package Microsoft.AspNetCore.SignalR.Client which depends on Microsoft.AspNetCore.SignalR.Client.Core and Microsoft.AspNetCore.Sockets.Client.Http

### CODE REVIEW

**Comment by muratg** [comment]
Reminder: you'll need to edit a couple more repos (e.g. coherence/packages.csv)

**Comment by moozzyk** [comment]
Thanks for the reminder. I will do that once the PR is approved but before merging.

File: src/Microsoft.AspNetCore.SignalR.Client/Microsoft.AspNetCore.SignalR.Client.csproj [hunk 1]
```diff
@@ -1,21 +1,13 @@
...
-  </ItemGroup>
-
-  <ItemGroup>
-    <ProjectReference Include="..\Microsoft.AspNetCore.SignalR.Common\Microsoft.AspNetCore.SignalR.Common.csproj" />
-    <ProjectReference Include="..\Microsoft.AspNetCore.Sockets.Abstractions\Microsoft.AspNetCore.Sockets.Abstractions.csproj" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="$(AspNetCoreVersion)" />
+    <ProjectReference Include="..\..\src\Microsoft.AspNetCore.SignalR.Client.Core\Microsoft.AspNetCore.SignalR.Client.Core.csproj" />
```
Comment (BrennanConroy) [inline src/Microsoft.AspNetCore.SignalR.Client/Microsoft.AspNetCore.SignalR.Client.csproj]: Just `..\`

**Comment by BrennanConroy** [comment]
You'll have another PR with the new package and easier HubConnection?

**Comment by moozzyk** [comment]
The "new" package is here (Microsoft.AspNetCore.SignalR.Client). I am looking into simpler ways of instantiating the client but it will be a separate PR.

**Pull request merged**